src/Engine/Network/UDPClient.cpp

```diff
@@ -18,22 +17,35 @@ bool UDPClient::Connect(std::string playerName, std::string address, int port)
     m_ReceiverEndpoint = udp::endpoint(boost::asio::ip::address().from_string(address), port);
     m_Socket = boost::shared_ptr<boost::asio::ip::udp::socket>(new boost::asio::ip::udp::socket(m_IOService));
     m_Socket->open(boost::asio::ip::udp::v4());
+    boost::asio::socket_base::receive_buffer_size option(819200);
```

Comment:
Don't like the magic number here. What is that even? Maybe a define or const int.